//
//  Constants.swift
//  Realm Object Editor
//
//  Created by Ahmed Ali on 1/22/15.
//  Copyright (c) 2015 Ahmed Ali. All rights reserved.
//

import Foundation

let EntityName = "<!EntityName!>"
let ParentName = "<!ParentName!>"
let AttrName = "<!AttrName!>"
let AttrType = "<!AttrType!>"
let Seperator = "<!Seperator!>"
let Annotations = "<!Annotations!>"
let CapitalizedAttrName = "<!CapitalizedAttrName!>"
let AttrDefaultValue = "<!AttrDefaultValue!>"
let QuotedValue = "<!QuotedValue!>"
let IndexedAttributes = "<!IndexedPropertiesReplacement!>"
let IgnoredAttributes = "<!IgnoredPropertiesReplacement!>"
let DefaultValues = "<!DefaultValuesReplacement!>"
let RelationshipName = "<!RelationshipName!>"
let RelationshipType = "<!RelationshipType!>"
